src/libMediator/Mediator.cpp

```diff
@@ -93,4 +92,17 @@ void Mediator::UpdateTxBlockRand(bool isGenesis)
         randVec = sha2.Finalize();
         copy(randVec.begin(), randVec.end(), m_txBlockRand.begin());
     }
-}
\ No newline at end of file
+}
+
+std::string Mediator::GetNodeMode(const Peer& peer)
+{
+    std::lock_guard<mutex> lock(m_mutexDSCommitteeNetworkInfo);
+    if (peer == m_DSCommitteeNetworkInfo[0])
+    {
+        return "DSLD";
+    }
+    else
+    {
+        return "SHRD";
+    }
```

Comment:
return (peer == m_DSCommitteeNetworkInfo[0]) ? "DSLD" : "SHRD";

Reply:
There might be some other option in the future